impl/src/main/java/no/vegvesen/nvdb/sosi/encoding/charset/SosiCharsetEncoder.java

```diff
@@ -43,16 +43,23 @@
 abstract class SosiCharsetEncoder extends CharsetEncoder {
 
     private List<Character> chars;
+    private Character rangeMax;
+    private byte repl = (byte)'?';
```

Comment:
Bare kall den for noe ordentlig når du først har den i luften. replacementCodePoint eller replacementCharCode eller noe sånt. Vet at resten inni her ser ut som om det har rømt fra C. Skulle sikkert refaktorert hele greien.

Reply:
Done.